jwk/manager_redis.go

```diff
@@ -0,0 +1,128 @@
+package jwk
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ory-am/hydra/pkg"
+	"github.com/pkg/errors"
+	"github.com/square/go-jose"
+	"gopkg.in/redis.v5"
+)
+
+type RedisManager struct {
+	DB        *redis.Client
+	Cipher    *AEAD
```

Comment:
It doesn't look like you make any use of the Cipher? The idea is that JWKs are encrypted with the SYSTEM_SECRET and then stored in the datastore, reducing the likelihood of compromise.